rest/config.go

```diff
@@ -478,6 +478,9 @@ func readFromPath(path string, insecureSkipVerify bool) (rc io.ReadCloser, err e
 
 func (dbConfig *DbConfig) AutoImportEnabled() (bool, error) {
 	if dbConfig.AutoImport == nil {
+		if !dbConfig.UseXattrs() {
```

Comment:
I think the correct revert is to check for `UseXattrs()` in `StartOnlineProcesses`. At least, that is what unintentionally changed in the referenced commit.

https://github.com/couchbase/sync_gateway/blob/76613c439fb90640379aab667324791689df32a7/db/database.go#L641-L642

https://github.com/couchbase/sync_gateway/blob/22ffb9012391f5c70840570f264d562955f4836e/db/database.go#L2220-L2221

Reply:
I simplified this code which forces user to pass in correct value, which tests do, and we validate the value for customers in https://github.com/couchbase/sync_gateway/blob/8bc18bd35833deea85b6f21586f0b8de7059a0f4/rest/config.go#L668